CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
+## 2.0.3
+  - Declare plugin as not worker safe for both server and client mode. (IO#select is not threadsafe)
```

Comment:
The server mode was already marked as `workers_not_supported` before my change. Is this not the case? It shouldn't be, AFAICT given that only one worker can bind to a port.